app/zpt/handler/atmos.py

```diff
@@ -0,0 +1,69 @@
+import numpy as np
+from scipy.integrate import odeint  # type: ignore
+from scipy.interpolate import BSpline, splrep  # type: ignore
+
+from .geos import g  # type:ignore
+
+AVOGADRO = 6.02282e23  # [mol^-1] aovogadros number
```

Comment:
```suggestion
AVOGADRO = 6.02214076e23   # [mol^-1] aovogadros number
```
This actually has an exact definition these days, so no reason not to use it: https://www.nist.gov/si-redefinition/meet-constants